docs/topics/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -0,0 +1,64 @@
+[#proc_deploying-an-application-to-openshift]
+= Deploying an Application to OpenShift
+
+.Prerequisites
+
+* The `oc` CLI client installed.
+* Maven installed.
```

Comment:
You also need the application.

Reply:
Yeah, that makes sense:) thank you for pointing that out.